js/ui/autosuggest-data.js

```diff
@@ -906,3 +908,5 @@ window.ScratchpadAutosuggestData = {
             "keyPressed", "keyReleased", "keyTyped"]
     },
 };
+
+export default AutosuggestData;
```

Comment:
You mentioned that we had turned of auto-suggest... do we still need to the files or are you planning on re-enabling and/or making it configurable in the future?